docs/reference.cue

```diff
@@ -351,16 +353,16 @@ _values: {
 		relevant_when?: string
 		required:       bool
 		type: {
-			"*": {}
-			"[string]"?: {
+			{"*": {}} |
```

Comment:
I think this will end up with a big oops unless `closed`. The evaluation will assume all types satisfy the values here, and rightfully so, since they *can be expanded* to satisfy the value, as they're not closed.

Reply:
What do you mean exactly?

Reply:
If it works - nevermind :D